registries/registry.go

```diff
@@ -166,24 +165,8 @@ func (r Registry) RegistryName() string {
 }
 
 // NewRegistry - Create a new registry from the registry config.
-func NewRegistry(con *config.Config, asbNamespace string) (Registry, error) {
+func NewRegistry(configuration Config, asbNamespace string) (Registry, error) {
```

Comment:
I hate to do the rename thing. Just curious if we should make this `RegistryConfig` to match some of the others, like `EtcdConfig` and `ClusterConfig`.

Reply:
Maybe better as a follow-on issue because I think this is used in more than just `registry.go`

Reply:
This is just `Config` because it belongs to a pkg called registries, so when written out you don't end up with stutter: `registries.RegistryConfig`. It's a go convention I'm not fond of for exactly your reason, but I believe I've had linters yell at me for it.